.github/actions/people/people.py

```diff
@@ -0,0 +1,613 @@
+"""Use the github API to get lists of people who have contributed in various ways to Pydantic.
+
+This logic is inspired by that of @tiangolo's
+[FastAPI people script](https://github.com/tiangolo/fastapi/blob/master/.github/actions/people/app/main.py).
+"""
+import requests
+import logging
+import subprocess
+import sys
+from collections import Counter
+from datetime import datetime, timedelta, timezone
+from pathlib import Path
+from typing import Any, Container, Dict, List, Set, Union
+
+import yaml
+from github import Github
+from pydantic import BaseModel, SecretStr
+from pydantic_settings import BaseSettings
+
+github_graphql_url = "https://api.github.com/graphql"
+
+discussions_query = """
+query Q($after: String) {
+  repository(name: "pydantic", owner: "samuelcolvin") {
```

Comment:
Shouldn't the owner be `pydantic` as well?
```suggestion
  repository(name: "pydantic", owner: "pydantic") {
```

Reply:
I thought so too but if I recall, the query doesn't work when I do that 🤷‍♀️.